### ISSUE TITLE
Cannot visit Theia server started in Docker

### ISSUE THREAD

**Issue by dclong** [theia-ide/theia-apps#45]
I launched a container using the latest version of the offical Theia Docker on a server, but cannot visit it. I have no problem to visit it if I launch a container locally (on my Mac). Does the Theia Docker miss some configuration to make it usable by all ips?

**Comment by marcdumais-work** [comment]
Hi @dclong 

Are you able to remotely access other containers that are running on that server? My guess is that a docker network configuration might be required when starting the container. That configuration is very context-specific, and would need to be crafted by you. See for example:

https://stackoverflow.com/questions/40898238/connect-to-container-from-a-remote-machine

**Comment by dclong** [comment]
@marcdumais-work 

Yes, I'm able to remotely access other contains running on the server. Actually my description wasn't precise. When I visit the Theia server, it freezes at the page with flashing dots (see the screenshot below). 
![image](https://user-images.githubusercontent.com/824507/42284797-28df99a0-7fe0-11e8-9b11-19158accbc9f.png)

**Comment by marcdumais-work** [comment]
@dclong that backend trace looks normal. Can you look in the browser's dev tools for any console error messages? 

Also can you confirm which image you are using? Is it the following one and if so which version/tag?   https://hub.docker.com/r/theiaide/theia/
